packages/apps/human-app/server/src/modules/statistics/spec/statistics.service.spec.ts

```diff
@@ -64,37 +67,43 @@ describe('StatisticsService', () => {
   describe('getOracleStats', () => {
     it('should return cached data if available', async () => {
       const cachedData = { some: 'data' };
+      const oracleCacheKey = EXCHANGE_ORACLE_ADDRESS + TOKEN;
       cacheManager.get.mockResolvedValue(cachedData);
 
-      const command: OracleStatisticsCommand = { address: EXCHANGE_ORACLE_ADDRESS };
+      const command: OracleStatisticsCommand = {
+        address: EXCHANGE_ORACLE_ADDRESS,
+        token: TOKEN,
+      };
       const result: OracleStatisticsResponse =
         await service.getOracleStats(command);
 
-      expect(cacheManager.get).toHaveBeenCalledWith(command.address);
+      expect(cacheManager.get).toHaveBeenCalledWith(oracleCacheKey);
       expect(result).toEqual(cachedData);
       expect(exchangeGateway.fetchOracleStatistics).not.toHaveBeenCalled();
     });
 
     it('should fetch, cache, and return new data if not in cache', async () => {
       const newData = { newData: 'data' };
+      const oracleCacheKey = EXCHANGE_ORACLE_ADDRESS + TOKEN;
```

Comment:
may be defined closer to the usage

Reply:
Issue: 1ed59b1b Reverted ("Add token authentication for Oracle statistics.")